target/scripts/build/compile.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# -eE :: exit on error (do this in functions as well)
+# -u  :: show (and exit) when using unset variables
+# -o pipefail :: exit on error in pipes
+set -eE -u -o pipefail
+
+# shellcheck source=../helpers/log.sh
+source /usr/local/bin/helpers/log.sh
+
+_log_level_is 'trace' && QUIET='-y' || QUIET='-qq'
```

Comment:
This is fixed as the `LOG_LEVEL` arg is missing in the stage running this script.